bundles/org.openhab.binding.jablotron/README.md

```diff
@@ -0,0 +1,126 @@
+| bridge               | lang                  | the language of the alarm texts                                                    | optional, default=en  |
+| JA-80/JA-100/JA-100F | refresh               | the channels refresh period in seconds                                             | optional, default=60s |
+| JA-80/JA-100/JA-100F | serviceId             | the service ID which identifies the alarm                                          | mandatory             |
+| JA-100/JA-100F       | code                  | the master code for controlling sections                                           | optional              |
+
+## Channels
+
+| thing                | channel name     | description                                               |
+|----------------------|------------------|-----------------------------------------------------------|
+| bridge               | N/A              | the bridge does not expose any channels                   |
+| JA-80                | statusA          | the status of the A section                               |
+| JA-80                | statusB          | the status of the AB/B section                            |
+| JA-80                | statusABC        | the status of the ABC section                             |
+| JA-80                | statusPGX        | the status of PGX                                         |
+| JA-80                | statusPGY        | the status of PGY                                         |
+| JA-80                | command          | the channel for sending keyboard codes to the OASIS alarm |
+| JA-80/JA-100/JA-100F | lastEvent        | the text description of the last event                    |
+| JA-80/JA-100/JA-100F | lastEventClass   | the class of the last event - e.g. arm, disarm, ...       |
+| JA-80/JA-100/JA-100F | lastEventInvoker | the invoker of the last event                             |
+| JA-80/JA-100/JA-100F | lastEventTime    | the time of the last event                                |
+| JA-80/JA-100/JA-100F | lastCheckTime    | the time of the last checking                             |
+| JA-80/JA-100/JA-100F | alarm            | the alarm trigger, might fire ALARM or TAMPER events      |
+| JA-100/JA-100F       | lastEventSection | the section of the last event                             |
+| JA-100               | STATE_%nr%       | the section %nr% status/control                           |
+| JA-100               | PGM_%nr%         | the PG section %nr% status/control                        |
+| JA-100               | THERMOMETER_%nr% | the thermometer %nr% value                                |
+| JA-100               | THERMOSTAT_%nr%  | the thermostat %nr% value                                 |
+| JA-100F              | SEC-%nr%         | the section %nr% status/control                           |
+| JA-100F              | PG-%nr%          | the PG section %nr% status/control                        |
```

Comment:
these dynamically created channels 1:1 match the internal alarm representation, which simplifies the overall binding functionality. Changing it is very dangerous since I cannot test it, I won't change these channel names.

Reply:
I don't think it adds much complexity. Some `.toLowerCase()` should be sufficient.

Reply:
not only .toLowerCase() unfortunately
I changed it, but cannot test it because I do no have JA100/f